lib/takeaway.rb

```diff
@@ -0,0 +1,39 @@
+class Menu
+
+  attr_reader :items
+
+  def initialize
+    @items = {
+      chicken: 5.0,
+      pizza: 6.5,
+      burrito: 4.0,
+      nachos: 3.5,
+      chips: 1.5
+    }
+  end
+
+  def display_menu
+    @items.collect do |k, v|
+      "#{k}, £#{v}0"
+    end
+  end
+end
+
+class Order
+
+  attr_reader :basket, :main_order
```

Comment:
I'm not sure I understand what `@main_order` is for. I can see that you use it to create a hash to put in `@basket`. But I can't see why (a) it needs to be an instance variable and (b) it needs to be exposed to things outside the class with an attribute reader.